src/llscan.cc

```diff
@@ -0,0 +1,470 @@
+      }
+
+      llnode::v8::JSFunction constructor(constructor_obj);
+
+      return constructor.Name(err);
+  }
+
+  if (type == v8->types()->kHeapNumberType) {
+    return "(HeapNumber)";
+  }
+
+  if (type == v8->types()->kJSArrayType) {
+    return "(Array)";
+  }
+
+  if (type == v8->types()->kOddballType) {
+    return "(Oddball)";
+  }
+
+  if (type == v8->types()->kJSFunctionType) {
+    return "(Function)";
+  }
+
+  if (type == v8->types()->kJSRegExpType) {
+    return "(RegExp)";
+  }
+
+  if (type < v8->types()->kFirstNonstringType) {
+    return "(String)";
+  }
+
+  if (type == v8->types()->kFixedArrayType) {
+    return "(FixedArray)";
+  }
+
+  if (type == v8->types()->kJSArrayBufferType) {
+    return "(ArrayBuffer)";
+  }
+
+  if (type == v8->types()->kJSTypedArrayType) {
+    return "(ArrayBufferView)";
+  }
+
+  if (type == v8->types()->kJSDateType) {
+    return "(Date)";
+  }
+
+  std::string unknown("unknown: ");
+
+  return unknown += std::to_string(type);
+}
+
+bool ListInstancesCmd::DoExecute(SBDebugger d, char** cmd,
+        SBCommandReturnObject& result) {
+    SBTarget target = d.GetSelectedTarget();
+    if (!target.IsValid()) {
+        result.SetError("No valid process, please start something\n");
+        return false;
+    }
+
+    v8::Value::InspectOptions inspect_options;
+
+    inspect_options.detailed = detailed_;
+
+    char** start = ParseInspectOptions(cmd, &inspect_options);
+
+    std::string full_cmd;
+    for (; start != nullptr && *start != nullptr; start++) full_cmd += *start;
+
+    SBExpressionOptions options;
+    SBValue value = target.EvaluateExpression(full_cmd.c_str(), options);
+    if (value.GetError().Fail()) {
+        SBStream desc;
+        if (!value.GetError().GetDescription(desc)) return false;
+        result.SetError(desc.GetData());
+        return false;
+    }
+
+    // Load V8 constants from postmortem data
+    llv8.Load(target);
+
+    v8::Value v8_mapaddr(&llv8, value.GetValueAsSigned());
+    v8::Error err;
+
+    if (err.Fail()) {
+        result.SetError("Failed to evaluate expression");
+        return false;
+    }
+
+    uint64_t mapaddr = value.GetValueAsUnsigned();
+    std::map<uint64_t, TypeRecord*>::iterator instance_it =
+            mapstoinstances->find(mapaddr);
+    if (instance_it != mapstoinstances->end()) {
+        TypeRecord* t = instance_it->second;
+        for (std::set<uint64_t>::iterator it = t->instances->begin();
+                it != t->instances->end(); ++it) {
+            v8::Error err;
+            v8::Value v8_value(&llv8, *it);
+            std::string res = v8_value.Inspect(&inspect_options, err);
+            result.Printf("%s\n", res.c_str());
+        }
+
+    } else {
+        result.Printf("No objects found with map %" PRIx64 "\n", mapaddr);
+    }
+
+    return true;
+
+}
+
+FindJSObjectsVisitor::FindJSObjectsVisitor(SBTarget* target,
+        SBCommandReturnObject* result) {
+    _target = target;
```

Comment:
Nitpicking again, sorry! Convention is slightly different in this project. We use `target_` with `_` at the end instead. May I ask you to fix this?